client/app/scripts/superdesk-packaging/packaging.js

```diff
@@ -467,7 +467,7 @@
             link: function(scope) {
                 scope.data = null;
                 scope.error = null;
-                scope.type = scope.item.type || scope.item.itemClass.split(':')[1];
```

Comment:
So Reuters objects have this property? Is it better to then leave this line as is (with additional checks for undefined) and update the `type` that is used in the template when we fetch data from server?
(if `item.location` is defined)